example_code/shmem_add_example.c

```diff
@@ -14,9 +14,10 @@ int main(void)
    shmem_barrier_all();
 
    if (me == 1){
-      old = shmem_add(&dst, 44, 0);
+      old = shmem_fadd(&dst, 44, 0);
```

Comment:
is this fadd or add example ? if this is fadd example the file has to be renamed